draft-ietf-mls-extensions.md

```diff
@@ -483,6 +846,93 @@ The `media_type` MAY be zero length, in which case, the media type of the
 `application_content` is interpreted as the first MediaType specified in
 `required_media_types`.
 
+## Role-Based Access Control Extension
```

Comment:
Could we please make this PR exclusive to safe extensions, and split-out a separate PR for RBAC (which I also support)?

Reply:
The RBAC extensions was meant as an example for use of the safe API. I'd be happy to move it to a separate PR. @psyoptix any thoughts?